macOS/DuckDuckGo/Preferences/Model/VPNPreferencesModel.swift

```diff
@@ -77,7 +78,7 @@ final class VPNPreferencesModel: ObservableObject {
     }
 
     private var isExclusionsFeatureAvailableInBuild: Bool {
-        proxySettings.proxyAvailable
+        vpnAppState.isUsingSystemExtension
```

Comment:
Refining this to whether the user is using a system extension.  For safety though I'll change this to always be true for non-APPSTORE builds.

Reply:
Change done.